### ISSUE TITLE
[Suggestion] DevOps Roadmap

### ISSUE THREAD

**Issue by dmartin-isp** [kamranahmedse/developer-roadmap#2224]
I feel like this roadmap is a bit out of date.  A lot of these early-mid skills are things I rarely ever use anymore.  Who uses strace or tcpdump anymore?  Everything is so dockerized and kubernetesified that we just blow up and rebuild everything so quickly, we don't really dive deep into the minutia of how stuff works or fails as much as we just build things out and try stuff until it works or doesn't.  Maybe I'm in a weird space, but that's my observation.

Awesome guides, by the way.  You've helped a lot of people.

**Comment by Yashrajput7232** [comment]
Yes DEVOPS updated road map should be a part of this  repo and it should involve docker and kubernities.

**Comment by ArpitKRS** [comment]
Can I do this??

**Comment by souravsasmal01** [comment]
Hey there, can you assign this issue to me?
